### ISSUE TITLE
go get unsuccessful 

### ISSUE THREAD

**Issue by avemeva** [yandex-cloud/docker-machine-driver-yandex#10]
Encountered this yesterday, any thoughts on how to avoid it? @GennadySpb
   
```
~ » go get -u github.com/yandex-cloud/docker-machine-driver-yandex                                                                                    
go: downloading github.com/yandex-cloud/docker-machine-driver-yandex v0.0.0-20210210140807-f5330e228f46
go: downloading github.com/yandex-cloud/go-genproto v0.0.0-20201102102956-0c505728b6f0
go: downloading github.com/yandex-cloud/go-sdk v0.0.0-20201109103511-a86298d3fea5
go: downloading github.com/yandex-cloud/go-sdk v0.0.0-20210211095836-1e646debd48a
go: downloading github.com/yandex-cloud/go-genproto v0.0.0-20210301094436-7f3b899d605e
go get: github.com/Sirupsen/logrus@none updating to
	github.com/Sirupsen/logrus@v1.8.0: parsing go.mod:
	module declares its path as: github.com/sirupsen/logrus
	        but was required as: github.com/Sirupsen/logrus
```

**Comment by avemeva** [comment]
Note: `go install` works fine

**Comment by GennadySpb** [comment]
Hi @andrewshvv ! 
Hm, interesting case. 
1) logrus dep used in ... `github.com/docker/engine/pkg/term/windows`
```
± go mod  why  github.com/sirupsen/logrus
# github.com/sirupsen/logrus
github.com/yandex-cloud/docker-machine-driver-yandex/driver
github.com/docker/machine/libmachine/ssh
github.com/docker/docker/pkg/term
github.com/docker/docker/pkg/term/windows
github.com/sirupsen/logrus
```
2) successful completion of the "go get" operation depends on the version of the go. 
1.14 - ok
1.15 - ok 
1.16 - fail

**Comment by GennadySpb** [comment]
/* Slowpoke mode ON */

Hi! Fixed in #14 

/* Slowpoke mode OFF */
